bitnami/ghost/Chart.yaml

```
@@ -40,4 +40,4 @@ maintainers:
 name: ghost
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/ghost
-version: 20.0.11
+version: 20.0.12
```

I think we will need to bump the version again
```suggestion
version: 20.0.13
```